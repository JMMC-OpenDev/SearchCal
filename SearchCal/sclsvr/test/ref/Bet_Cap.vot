version https://git-lfs.github.com/spec/v1
oid sha256:f36831dc281e987f250ab752f78325dbbc34cca077ac329340cdc6b7f3371b2f
size 76406
