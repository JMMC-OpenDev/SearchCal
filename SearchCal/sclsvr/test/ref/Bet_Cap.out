version https://git-lfs.github.com/spec/v1
oid sha256:c0285638aacc06c0bcb951f73622485600f6b6a58d011f69e505605d25b10c74
size 181432
